### PR TITLE
Remove virtual scroll list

### LINKED ISSUE #1887: [Enhancement] Remove List Virtual Scroll
- [X] I have written a [good issue](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Issue) 

## Describe the enhancement
We've decided to remove virtual scroll from Kirby (see #1883). 

Therefore everything to [list virtual scroll](https://cookbook.kirby.design/#/home/showcase/list-virtual-scroll) should be removed. 

## Describe the solution you'd like
Remove: 
- Cookbook page 
- Example components 
- Exported types 
- Code related to virtual scroll 
<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/master/.github/CONTRIBUTING.md/#the-process-of-contributing) correctly.

### Refinement

- [ ] Request that the issue is [UX refined](https://github.com/kirbydesign/designsystem/blob/master/.github/CONTRIBUTING.md/#ux-refinement); do not proceed until this is done.
- [ ] Request that the issue is [tech refined](https://github.com/kirbydesign/designsystem/blob/master/.github/CONTRIBUTING.md/#tech-refinement); do not proceed until this is done.

### Implementation 
The contributor who wants to implement this issue should: 

- [ ] Make sure you have read: "[Before you get coding](https://github.com/kirbydesign/designsystem/blob/master/.github/CONTRIBUTING.md/#before-you-get-coding)".
- [ ] Signal to others you are working on the issue by assigning yourself.
- [ ] Create a branch from the [master branch](https://github.com/kirbydesign/designsystem/tree/master) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Publish a WIP implementation to Github as a draft PR and ask for feedback. 
- [ ] Make sure you have implemented tests following the guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)".
- [ ] Update the [cookbook](https://cookbook.kirby.design) with examples and showcases.

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request. If you created a draft PR during implementation you can just mark that as "ready for review".

### PR DESCRIPTION
**Pull request by RasmusKjeldgaard**
## Which issue does this PR close?

This PR closes #1887

## What is the new behavior?
Virtual scroll is removed from documentation as well as the list implementation itself.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No (as the change virtual scroll list experimental)

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/master/.github/CONTRIBUTING.md/#the-process-of-contributing) correctly.

### Reminders
- [ ] Make sure you have implemented tests following the guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)".
- [ ] Make sure you have updated the cookbook with examples and showcases (for bug fixes, enhancements & new components).

### Review  
- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review).
- [ ] Request that the changes are code-reviewed 
- [ ] Request that the changes are [UX reviewed](https://github.com/kirbydesign/designsystem/blob/master/.github/CONTRIBUTING.md/#ux-review) (only necessary if your PR introduces visual changes)

When the pull request has been approved it will be automatically merged to master via [automerge](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/automatically-merging-a-pull-request).

### CODE REVIEW

File: libs/designsystem/src/lib/components/list/list.component.ts [hunk 1]
```diff
@@ -84,63 +81,6 @@ export class ListComponent implements OnInit, AfterViewInit, OnChanges {
   @Input()
   hasItemSpacing: boolean;
 
-  @Input() useVirtualScroll = false;
```
Comment (MadsBuchmann) [inline libs/designsystem/src/lib/components/list/list.component.ts:87]: Not related to these lines. 

But in the original PR (#1497) a bunch of comments detailing the role of each input property was removed. 
None of the input properties are well documented in the cookbook. 

So now that we have the file open and we're reverting a bit - should we consider to throw them back in for our own sake? Then we don't have to guess what each input property does based on naming when rebuilding declaratively 😄

[list.component.ts from #1497](
https://github.com/kirbydesign/designsystem/pull/1497/files#diff-98f909a9aa2931c335d0d4cebf4367d6e52652656209d67b1cb21a10fd07024)

**Review by MadsBuchmann — CHANGES_REQUESTED** [review]
Cross referenced with #1497 and it looks good! 

Left two minor comments, the one relating to package.json is the only blocker 😄

File: libs/designsystem/package.json [hunk 2]
```diff
@@ -10,7 +10,6 @@
     "date-fns-tz": "1.1.4",
     "highcharts": "8.0.4",
     "@fontsource/roboto": "4.2.1",
-    "ngx-ui-scroll": "^2.0.1",
```
[the remark this answers is not in the document]
Comment (MadsBuchmann) [reply libs/designsystem/package.json:13]: Also remember to run a `npm install` and commit the resulting `package-lock.json`.

[the remark this answers is not in the document]
Comment (RasmusKjeldgaard) [reply libs/designsystem/package.json:13]: Good catch, how could I forget that 😅

File: libs/designsystem/src/lib/components/list/list.component.ts (hunk 1 above)
Comment (RasmusKjeldgaard) [reply libs/designsystem/src/lib/components/list/list.component.ts:87]: Excellent suggestion, will do!

**Review by MadsBuchmann — APPROVED** [review]

**Pull request merged**